### PR TITLE
Feature/actvity contact

### PR DESCRIPTION
**Pull request by tuj**
Added contact form to activity page

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 ## In develop
 
 * Fixed translation issues with register page, and activity view.
-* Added Webmaster role and changed Editor permissions
+* Added Webmaster role and changed Editor permissions.
 * Fixed styling of cookie message and link.
+* Added contact form to activities.
 * Enabled Metatag: Open Graph module and added tags
```
Comment (magnify) [inline CHANGELOG.md]: Missing punctuation?

**Review by magnify — COMMENTED** [review]
Assigning to @cableman for review.

File: config/sync/user.role.anonymous.yml [hunk 2]
```diff
@@ -11,4 +11,3 @@ is_admin: false
 permissions:
   - 'access content'
   - 'view the administration theme'
-  - 'view sp metadata'
```
Comment (magnify) [inline config/sync/user.role.anonymous.yml]: Should this be removed?

File: web/modules/custom/itk_activity/itk_activity.info.yml [hunk 3]
```diff
@@ -2,4 +2,4 @@ name: ITK Activity
 type: module
 description: 'Supplies activity.'
 version: 1.0.0-alpha1
-core: 8.x
\ No newline at end of file
+core: 8.x
```
Comment (magnify) [inline web/modules/custom/itk_activity/itk_activity.info.yml:5]: Missing new line.

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/activity-sidebar.twig [hunk 4]
```diff
@@ -1,3 +1,4 @@
+{# @TODO: Refactor #}
```
Comment (magnify) [inline web/themes/custom/genlyd_aarhus/source/_patterns/02-components/activity-sidebar.twig]: Do we want this @TODO here?

**Review by magnify — CHANGES_REQUESTED** [review]
@mikkelboesen we should consider getting @cableman to take a look at this too.

File: web/modules/custom/itk_activity/itk_activity.info.yml (hunk 3 above)
Comment (tuj) [reply web/modules/custom/itk_activity/itk_activity.info.yml:5]: Not missing

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/activity-sidebar.twig (hunk 4 above)
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/source/_patterns/02-components/activity-sidebar.twig]: Fixed

File: config/sync/user.role.anonymous.yml (hunk 2 above)
Comment (tuj) [reply config/sync/user.role.anonymous.yml]: Fixed

**Review by magnify — APPROVED** [review]

**Pull request merged**